src/api/error.ts

```diff
@@ -61,6 +64,13 @@ export default class APIError extends CLIError {
     ];
   }
 
+  static paymentRequired(error: Error): MessagesAndExitCode {
+    const genericMessage =
+      error.message || 'You need to upgrade to a paid plan to perform this request.';
+
+    return [[genericMessage], 104];
```

Comment:
I had a convention to substract 300 to the http code status (404-300 = `104` CLI error, 422-300 = `122` CLI error), so for a 402 HTTP status I'd suggest a `102` CLI error.

There's no particular reason, nor suggestion for the rest of the world about this. I decided this convention by myself for “fun”. You can totally ignore this but if you make the change it would keep the exit codes coherent together (but completely incoherent to the rest of the world :stuck_out_tongue:)